logfire/experimental/api_client.py

```
@@ -509,6 +499,34 @@ def _handle_response(self, response: Response, *, is_case_endpoint: bool = False
+        custom_report_evaluator_types: Sequence[type[Any]],
+    ) -> Dataset[InputsT, OutputT, MetadataT] | DatasetDetail | ExportedDataset:
+        data = self._handle_response(response)
+        if not include_cases:
+            return _validate_or_warn(_dataset_detail_adapter, data)
+        if input_type is None:
+            return _validate_or_warn(_exported_dataset_adapter, data)
+
+        try:
+            from pydantic_evals import Dataset
+        except ImportError:
+            raise ImportError('pydantic-evals is required for this operation. Install with: pip install pydantic-evals')
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)